lib/statement.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'record_list'
+
+class Statement
+
+  def initialize(record_list)
+    @list = record_list.get_list
+    display_transactions
+  end
+
+  def display_transactions
```

Comment:
Is there a reason `display_transactions` exists, given that it purely sends a message with a different name to the same object with no extra arguments?